esp/esp/program/controllers/lottery.py

```diff
@@ -290,6 +290,11 @@ def fill_section(self, si, priority=False, rank=10):
         
         #   Compute number of spaces - exit if section or program is already full.  Otherwise, set num_spaces to the number of students we can add without overfilling the section or program.
         num_spaces = self.section_capacities[si] - numpy.sum(self.student_sections[:, si])
+        # TODO(benkraft): This doesn't support program_size_by_grade.  For
+        # Stanford's use case, they don't need it -- they want the cap to
+        # apply to entering the lottery too.  But we should either decide that
+        # that's the Right Way and remove the program size logic from here
+        # entirely, or implement it here too.
```

Comment:
I'll add a warning for now, and update the comment a bit.  As currently implemented, the fact that the cap (if it exists) applies to entering the lottery is hard-coded, so this logic is useless, but it's possible that that could change in the future if there is demand to make that configurable.  My guess is there won't be -- I think not many programs use the cap as-is, and the fact that until now having "All student deadlines" open would cause it to be ignored supports that theory.